org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/Messagebox.java

```diff
@@ -353,6 +355,15 @@ public static int showDialog(String message, String title, int buttons, String i
 		Messagebox msg = new Messagebox();
 		return msg.show(message, title, buttons, icon, editor, callback, modal);
 	}
+	
+    // Andreas Sumerauer IDEMPIERE 4702
+	@Listen("onCancel = #form")
```

Comment:
Should be just @Listen("onCancel") here, MessageBox doesn't has an element with id="form".